README.md

```diff
@@ -46,21 +45,21 @@ To build the project and generate the bundle use Maven
 
     mvn clean package
 
-If everything checks out, the bundle for P2 should be available in the `target` folder
+If everything checks out, the bundle for p2 should be available in the `target` folder
```

Comment:
Trivial: This is not you, but this sentence doesn't end in a period.

Reply:
Done 026b33e37278fdd285655fb647acc45dde7c1143